test/PrismicLinkTestWrapper.svelte

```diff
@@ -0,0 +1,24 @@
+<script lang="ts">
+	// This file is a wrapper for the PrismicLink component to be used in tests as
+	// passing slot content to the PrismicLink component is not easy in tests.
+
+	import { FilledLinkToWebField } from "@prismicio/client";
+
+	type $$Props = {
+		field: FilledLinkToWebField;
+		children?: string;
+	};
+
+	export let field: $$Props["field"];
+	export let children: $$Props["children"] = undefined;
+
+	import { PrismicLink } from "../src";
+</script>
+
+{#if children}
+	<PrismicLink {field}>
+		{children}
+	</PrismicLink>
+{:else}
+	<PrismicLink {field} />
+{/if}
```

Comment:
💡 #idea: Nice solution! Here's how it could be cleaned up to be more idiomatic:

```svelte
<!--
This file is a wrapper for the PrismicLink component to be used in tests as
passing slot content to the PrismicLink component is not easy in tests.
-->

<script lang="ts">
	import { FilledLinkToWebField } from "@prismicio/client";
	import { PrismicLink } from "../src";

	export let field: FilledLinkToWebField;
	export let children: string | undefined = undefined;
</script>

{#if children}
	<PrismicLink {field}>{children}</PrismicLink>
{:else}
	<PrismicLink {field} />
{/if}
```

Reply:
Oh nice! Love it, done!